src/ansys/geometry/core/designer/face.py

```diff
@@ -196,6 +200,7 @@ def body(self) -> "Body":
         return self._body
 
     @property
+    @min_backend_version(24, 2, 0)
```

Comment:
This will raise an error if the version is lesser than 24.2 before launching the function right? I think you might never reach the except of the try in this function because of that

Reply:
The decorator is for the ``shape`` property. The try/except is on the ``normal`` function

Reply:
Misread the diff 😅

Reply:
Yeah the diff was a bit ugly - I know 😄